inc/saiobject.h

```diff
@@ -239,6 +239,50 @@ sai_status_t sai_bulk_get_attribute(
         _Inout_ sai_attribute_t **attr_list,
         _Inout_ sai_status_t *object_statuses);
 
+/**
+ * @brief Queries attributes of objects in bulk based on a query condition.
+ *
+ * This function performs a bulk query on SAI objects of a specified type,
+ * based on the provided query condition. It retrieves the keys of the
+ * objects that match the query condition, their attributes, and statuses.
+ *
```

Comment:
description is too vague, it does not explain why you need 2 sets of attr_cunt passed

can you give example how this api should work ?, because it seems like you want to do a generic bulk_get api for N number of objects which each one can have different attributes, we have bulk api for each specific objects already (no all all populated) but i think this could be added to each specific object type instead of making 1 generic api

Reply:
Thanks for your comments.
I relied here https://github.com/opencomputeproject/SAI/pull/1948#issuecomment-1907369064.

Reply:
> description is too vague, it does not explain why you need 2 sets of attr_cunt passed
> 
> can you give example how this api should work ?, because it seems like you want to do a generic bulk_get api for N number of objects which each one can have different attributes, we have bulk api for each specific objects already (no all all populated) but i think this could be added to each specific object type instead of making 1 generic api

Thanks for your comments.
I relied here https://github.com/opencomputeproject/SAI/pull/1948#issuecomment-1907369064.